backend/views/retrieve_test_view.py

```diff
@@ -4,13 +4,34 @@
 from custom_models.item_text import ItemText
 from custom_models.test import Test
 from custom_models.language import Language
+from custom_models.question import Question
+from custom_models.question_type import QuestionType
+from custom_models.item_type import ItemType
 
 TEST_META_DATA = "test_meta_data"
 TEST_INSTRUCTIONS = "test_instructions"
 TEST_QUESTIONS = "test_questions"
+TYPE = "type"

```

Comment:
I don't think type or map are still used?

Reply:
dropped in c1b0390